sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubConsumerIntegrationTest.java

```diff
@@ -229,7 +230,10 @@ public void multipleConsumers() {
      * Verify that we can receive until the timeout multiple times.
      */
     @Test
-    public void receiveUntilTimeoutMultipleTimes() {
+    public void receiveUntilTimeoutMultipleTimes() throws IOException {
+        this.consumer.close();
```

Comment:
Not sure why this is required but if you intended to close out the consumer that was created from previous tests, you can instead use this:

```
@After
public void cleanUp() {
    this.consumer.close();
    this.consumer = null;
}
```

Reply:
Oh, that was because I don't use the consumer created in beforeTest() and create my own. I'll remove it. It's not necessary.